test/jdk/java/foreign/TestMemoryHandleAsUnsigned.java

```diff
@@ -1,257 +0,0 @@
-/*
```

Comment:
Why was this file deleted?

Reply:
Looking closer, it seems that the `asUnsigned` combinator was removed altogether?

Reply:
Yes, we have decided to remove the combinator, at least for now. While the other combinators are very general, this one feels very specific, and I'm not 100% sure if it's worth the complexity. We can always re-add later.